package.json

```diff
@@ -370,6 +370,34 @@
       }
     ]
   },
+  "nyc": {
+    "extends": "@istanbuljs/nyc-config-typescript",
+    "require": [
+      "ts-node/register",
+      "source-map-support/register"
+    ],
+    "all": true,
+    "exclude": [
+      "node_modules/**",
+      "**/*.d.ts",
+      ".{github,nyc_output,vscode,vscode-test}",
+      "{media,infra}/**",
+      "coverage/**",
+      "src/Tests/**"
+    ],
+    "include": [
+      "src/**/*.ts",
+      "out/**/*.js"
+    ],
+    "instrument": true,
+    "reporter": [
+      "cobertura",
+      "lcov",
+      "html",
+      "text",
+      "text-summary"
+    ]
+  },
```

Comment:
Could you share any reference about this setting?

Reply:
This is for config of nyc
- https://github.com/istanbuljs/nyc#common-configuration-options